src/webob/acceptparse.py

```diff
@@ -376,8 +376,8 @@ def parse(cls, value):
 
         :param value: (``str``) header value
         :return: If `value` is a valid ``Accept-Language`` header, returns a
-                 generator that yields (language range, quality value) tuples,
-                 as parsed from the header from left to right.
+                 generator iterator that yields (language range, quality value)
```

Comment:
just "iterator" would be appropriate terminology here. The fact that it is a generator is not useful to the user.